### ISSUE TITLE
scriptengine: deprecated festures removed since v8-3.16.5

### ISSUE THREAD

**Issue by TheSiege** [desura/Desurium#450]
https://gist.github.com/TheSiege/4716709

```
[ 22%] Building CXX object src/static/scriptengine/CMakeFiles/scriptengine.dir/code/jsExtensions.cpp.o
cd /var/tmp/portage/games-util/desurium-9999/work/desurium-9999_build/src/static/scriptengine && /usr/bin/x86_64-pc-linux-gnu-g++   -DBOOST_FILESYSTEM_VERSION=3 -DBOOST_NO_SCOPED_ENUMS=1 -DNDEBUG -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES -D__WXGTK__ -DwxDEBUG_LEVEL=0 -O2 -pipe -march=native -ftree-vectorize -mavx -maes   -fPIC -pipe -fvisibility=hidden -finline-functions  -fpermissive -std=c++0x -I/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/include -I/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/common -I/var/tmp/portage/games-util/desurium-9999/work/desurium-9999_build/include -I/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine -I/var/tmp/portage/games-util/desurium-9999/work/desurium-9999_build/fetch_cef-prefix/src/fetch_cef    -o CMakeFiles/scriptengine.dir/code/jsExtensions.cpp.o -c /var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp: In member function 'virtual void* JSObject::getUserObject()':
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp:215:11: error: 'Unwrap' is not a member of 'v8::External'
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp: In member function 'virtual ChromiumDLL::JSObjHandle JSFactory::CreateObject(void*)':
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp:293:31: error: 'Wrap' is not a member of 'v8::External'
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp: In static member function 'static v8::Handle<v8::Value> JSExtension::FunctionCallbackImpl(const v8::Arguments&)':
/var/tmp/portage/games-util/desurium-9999/work/desurium-9999/src/static/scriptengine/code/jsExtensions.cpp:342:52: error: 'Unwrap' is not a member of 'v8::External'
make[2]: *** [src/static/scriptengine/CMakeFiles/scriptengine.dir/code/jsExtensions.cpp.o] Error 1
make[2]: Leaving directory `/var/tmp/portage/games-util/desurium-9999/work/desurium-9999_build'
make[1]: *** [src/static/scriptengine/CMakeFiles/scriptengine.dir/all] Error 2
make[1]: Leaving directory `/var/tmp/portage/games-util/desurium-9999/work/desurium-9999_build'
make: *** [all] Error 2
```

EDIT by @karolherbst:
/*\* Deprecated, use New instead. _/
V8_DEPRECATED(V8_INLINE(static Local<Value> Wrap(void_ value)));

/*\* Deprecated, use a combination of Cast and Value instead. _/
V8_DEPRECATED(V8_INLINE(static void_ Unwrap(Handle<Value> obj)));

TODO:
- [ ] update depends of ebuilds
- [ ] port to new API

**Comment by karolherbst** [comment]
these functions are gone with 3.16.5

**Comment by karolherbst** [comment]
this new API is so old, that it is even included in v8-1.2.0, so I am sure, that porting won't break anything

**Comment by karolherbst** [comment]
there were implemented like:

``` c
Local<Value> External::Wrap(void* value) {
  return External::New(value);
}

void* External::Unwrap(Handle<v8::Value> obj) {
  return External::Cast(*obj)->Value();
}
```
